### DISCUSSION TITLE
Nemo: Add a “mixed” sort option to mix files and folders

### DISCUSSION THREAD

**Discussion by AlexDaniel** [linuxmint/discussions#526] [Ideas]
Some file managers, like SpaceFM, allow “mixed” mode, where folders and files are mixed together. This is extremely useful in folders like “Downloads”, where you can sort everything based on the modification date and immediately see all closely related files, sorted by date. For example, if you have a new `.zip` file and you extract it, the extracted folder will sit right next to it.
![peek_1](https://github.com/user-attachments/assets/98d235bc-8e6f-4343-b3b7-676cc690a148)

**Comment by cplittlfield** [comment]
Have you tried turning off Preferences > Views > Default View > "Sort folders before files"?  That should do what you want.

**Comment by AlexDaniel** [comment]
Ahhh! There it is! Thanks!

Now if only I could decrease the spacing between the files, find a way to truncate long filenames, and also make the quick search jump to the file if I type any part of the name (not just the beginning), it'd be a perfect file manager. :) If you know solutions to any of the above, let me know!

**Reply by cplittlfield** [reply]
The slider bar at the bottom right of a Nemo window will adjust icon spacing, but it also adjusts icon size, as well, which may not be what you want.  Icon view and list view both display truncated file names, but I don't know any way to do that in compact view.  If you want to truncate file names permanently, however, the bulk-rename feature in Nemo (an app called bulky) can be helpful there.  The search feature enabled from the menu or the magnifying glass icon will match inside file names, just not the search feature accessed by simply typing.
